src/Configuration/PlatformConfigMapper.js

```diff
@@ -0,0 +1,28 @@
+import { createReadStream } from 'fs';
+import { parse } from 'csv-parse';
+
+export default class PlatformConfigMapper {
```

Comment:
Filename could be CSV parser. What about platform config are we speaking;

Reply:
_that came as part of a bit of over thinking for future_ 

e.g. if we have multiple platform - currently focusing on WooCommerce 
- we can also have support for json, yaml, or xml along with csv

hence keeping things generic - **Platform** (instead of woocommerce) **Config** (as we are taking configuration)
**Mapper** (for mapping)